ISI Web of Knowledge.js

```diff
@@ -2,189 +2,14 @@
 	"translatorID": "594ebe3c-90a0-4830-83bc-9502825a6810",
 	"label": "ISI Web of Knowledge",
```

Comment:
This should be renamed to something like "Web of Science Tagged Format", which is pretty long, but I wouldn't want to shorten it to WoS. Thoughts? CC @adam3smith